test/SnowballStemmer_test.dart

```diff
@@ -0,0 +1,48 @@
+// SPDX-License-Identifier: Apache-2.0
+
+import 'dart:collection';
+import 'dart:core';
+import 'dart:io';
+
+import '../lib/SnowballStemmer.dart';
+
+final testData = LinkedHashMap<String, String>();
+final testDataRegExp = RegExp(r'^([^\s]+)\s+([^\s]+)$');
+
+int passes = 0;
+int fails = 0;
+
+main() {
```

Comment:
These can be local inside main

```suggestion
main() {
  int passes = 0;
  int fails = 0;
```

Reply:
Fixed differently